packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -109,12 +118,21 @@ class DDSFooterComposite extends ModalRenderMixin(HybridRenderMixin(HostListener
   @property({ attribute: 'disable-locale-button' })
   disableLocaleButton = false;
 
+  /**
+   * `true` to enable language selector.
+   */
+  @property({ attribute: 'enable-language-selector' })
+  enableLanguageSelector = false;
+
   /**
    * The language used for query.
    */
   @property()
   language?: string;
 
+  @property({ attribute: false })
+  langList?;
```

Comment:
Does this serve a different purpose from `localeList`?

Reply:
Yes, it maps the `combo-box-item` elements to the language list that was added with this commit. The locale list has many repeated languages. See also my other comment.